Messenger/AppDelegate.mm

```diff
@@ -698,6 +698,11 @@ - (void)windowDidBecomeKey:(NSNotification*)notification {
   [self evaluateJavaScript:@"try { (typeof MacMessenger != 'undefined') && MacMessenger.focusComposer(); } catch(_) {}"];
 }
 
+- (void)windowDidResignKey:(NSNotification *)notification {
+  // Make sure a blur event is triggered when the app is on another workspace
```

Comment:
This is a misleading description — windowDidResignKey: is called every time the window is not the current user input target, like when switching to a different app in the same space or opening a different window in the same app (like the "about" window.)

If what you're looking to do is to emit a blur event when the window is on a different space (than the user's focus), you might want to look into leveraging different notifications and delegate methods.